Translated readme.md (#9)
ToucanToco/dev-manifesto: readme_en.md

# dev-manifesto

The Tokardy tech team appreciates and values the following points for (future-) Tokars:

## 1. Production / method

Being agile, which means:
- work on what brings the most value to Toucan at the moment t
  <details>
    <summary>Examples</summary>

    To find out, I ask myself the following questions:
    - will what I am doing benefit a user/customer/designer/tokar from the moment it is launched? If so, will it allow him to do something he couldn't do before? Will it allow him to do something twice as fast as before?
    - on the other hand, is what I have done a marginal improvement, or one that does not benefit anyone in the immediate future?
    - is what I'm working on merged/integrated into the product? Or is it dropped?
  </details>

> - set very progressive steps and stick to them

Reviewer: ```suggestion
- plan small simple steps and stick to them
```

  <details>
    <summary>Examples</summary>

>     - my feature has been divided, every bit of the feature is written in the Trello card, it is unique and each step is short

Reviewer: ```suggestion
    - the feature I work on has been divided into small simple steps, clearly described in a Trello card
```

>     - my feature was not divided, I quickly saw that it was more complicated than expected, I put the Trello card back into cut-out

Reviewer: ```suggestion
    - the feature I work on was not properly planned, I quickly saw that it was more complicated than expected, I send the Trello card back into the design column
```

  </details>
- deliver these small steps as often as possible
  <details>
    <summary>Examples</summary>

>     - I can make a v0 of my functionality and present it in a few days

Reviewer: ```suggestion
    - I can make a v0 of my feature and present it in a few days
```

>     - I can start a project with confidence that PR will be offered in the evening

Reviewer: ```suggestion
    - I can start a project with confidence that PR will be opened in the evening
```

>     - I focus on the essentials in the PRs and avoid making additional modifications "by the way".

Reviewer: ```suggestion
    - I focus on the essentials in the PRs and avoid making additional modifications, "drive by changes".
```

  </details>

> - communicate your goal and progress to the rest of the team

Reviewer: ```suggestion
- state your goal and communicate your progress to the rest of the team
```

  <details>
    <summary>Examples</summary>

>     - I communicate via daily on my current projects, my difficulties

Reviewer: ```suggestion
    - I use our daily meeting to communicate on my current projects, my difficulties
```

    - I update Trello, the wiki
    - my PRs are clear and self-sufficient (screenshots, test urls, examples of API use, examples of confs)

>     - I can retrospectively say the projects and tasks on which I have participated

Reviewer: ```suggestion
    - It's easy and enjoyable for me to speak about the projects and tasks on which I have contributed
```

  </details>

## 2. Autonomy

> Being autonomous means that when you take a subject, you commit yourself to it and put the necessary energy into it to get to the point of rendering it.

Reviewer: ```suggestion
Being autonomous means that when you start on a project, you commit yourself to it and put the necessary energy into it to deliver it.
```

This does not mean **NOT** working alone.

In particular, we recommend:

> - to precisely identify the need to which we are responding: what is the demo of my feature?

Reviewer: ```suggestion
- to identify the needs to which we are responding: what is the demo of my feature?
```

- to identify precisely the points on which assistance is needed
  <details>
    <summary>Example</summary>

>     - I know how to do my self-criticism when a task does not progress and I know how to communicate it

Reviewer: ```suggestion
    - I know how to ask for help when I'm stuck with a task
    - I know how to get support from other devs on difficult technical project, and show that they are worth it
```

  </details>
- to go to his colleagues with specific and clear questions

> - to take the tasks at the top of the common backlogs that I can take and commit to process

Reviewer: ```suggestion
- to start with the tasks at the top of the common backlogs, where I feel and I can deliver and  commit to process
```

## 3. Apprenticeship

> To be a volunteer and a driving force in your learning, and to participate in the learning of your colleagues, is:

Reviewer: ```suggestion
To be a driving force in continuous learning, for you, your colleagues and the organisation:
```

- take the time to peer-program as often as possible
  <details>
    <summary>Examples</summary>

    - Communicate well with the rest of the team to synchronize with your teammate by setting events in the calendar
  </details>

> - document during the learning process for the following

Reviewer: ```suggestion
- document your the learning process so that it cal help others
```

> - follow his learning (thanks to a board for example) and communicate it to the rest of the team, then take the lead on newly acquired subjects to validate them

Reviewer: ```suggestion
- follow up on what you have learned (use a Trello board for example) and communicate it to the rest of the team, take the lead on projects in your new areas of expertise to validate them
```

## 4. Attitude

Have a good work ethic, be honest with yourself and your colleagues about:

> - its skills

Reviewer: ```suggestion
- skills
```

> - its difficulties (which will be used to solve them)

Reviewer: ```suggestion
- difficulties
```

> - its ability to deliver (there is no point in announcing the moon, if not to disappoint)

Reviewer: ```suggestion
- our ability to deliver (there is no point in announcing the moon, if not to disappoint)
```
